staging/src/k8s.io/apiextensions-apiserver/pkg/controller/status/naming_controller.go

```diff
@@ -111,6 +111,7 @@ func (c *NamingConditionController) getAcceptedNamesForGroup(group string) (allR
 
 		allKinds.Insert(item.Status.AcceptedNames.Kind)
 		allKinds.Insert(item.Status.AcceptedNames.ListKind)
+		allKinds.Insert(item.Status.AcceptedNames.Categories...)
```

Comment:
categories are not kinds... why are we adding them here?

Reply:
Removed.